crates/oxc_linter/src/rules/eslint/no_fallthrough.rs

```diff
@@ -53,9 +56,18 @@ impl NoFallthrough {
         allow_empty_case: Option<bool>,
         report_unused_fallthrough_comment: Option<bool>,
     ) -> Self {
-        let comment_pattern = comment_pattern.unwrap_or(DEFAULT_FALLTHROUGH_COMMENT_PATTERN);
+        let custom_comment_pattern = if let Some(pattern) = comment_pattern {
+            Some(Regex::new(format!("(?iu){pattern}").as_str()).unwrap())
+        } else {
+            None
+        };
+
         Self(Box::new(Config {
-            comment_pattern: Regex::new(format!("(?iu){comment_pattern}").as_str()).unwrap(),
+            custom_comment_pattern,
+            default_comment_pattern: AhoCorasick::builder()
+                .ascii_case_insensitive(true)
+                .build(DEFAULT_FALLTHROUGH_COMMENT_PATTERNS)
+                .expect("Could not build AhoCorasick for default comment patterns"),
```

Comment:
Can we provide the name of the rule so users know where and how to fix their configs?